core/src/avm2/globals/flash/display3D/index_buffer_3d.rs

```diff
@@ -13,14 +12,45 @@ pub fn index_buffer_3d_allocator<'gc>(
 
 pub fn upload_from_byte_array<'gc>(
     activation: &mut Activation<'_, 'gc>,
-    _this: Option<Object<'gc>>,
-    _args: &[Value<'gc>],
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    avm2_stub_method!(
-        activation,
-        "flash.display3D.IndexBuffer3D",
-        "uploadFromByteArray"
-    );
+    if let Some(index_buffer) = this.and_then(|this| this.as_index_buffer()) {
+        let byte_array = args
+            .get(0)
+            .unwrap_or(&Value::Undefined)
+            .coerce_to_object(activation)?;
```

Comment:
Can we use `get_object` to correctly handle null values? We should start avoiding direct get-and-coerce